src/selectionHandler.ts

```diff
@@ -135,16 +140,18 @@ function eventDataOf(
   });
 
   const { length } = currentCompletion;
-  const netLength = editor.selection.anchor.character - position.character;
+  const netLength = length - (oldPrefix?.length || 0);
```

Comment:
this is the main change - `netLength` was calculated with positions, which is problematic if the completion takes more than one line. i changed it to be based on the completion parameters themselves (`old_prefix` is quite exactly the opposite of what we mean by `netLength`)

Reply:
this is also how we calculate it in the intellij plugin: https://github.com/codota/tabnine-intellij/blob/5c66c60f08d60463cf87b372264b3638c6faee1a/src/main/java/com/tabnine/selections/CompletionPreviewListener.java#L27